roles/apache-perun/templates/perun-ssl.conf.j2

```diff
@@ -78,6 +78,17 @@ ShibCompatValidUser on
  #RewriteCond %{REQUEST_URI} !^/Shibboleth.sso/
  #RewriteRule .* /maintenance/maintenance.html [R,L]
 
+ ####################################
+ ##    DEPLOY TO TOMCAT          ####
+ ####################################
+
+ <Location "/tomcat-manager/">
+   ProxyPass http://127.0.0.1:18443/ nocanon
+   ProxyPassReverse http://127.0.0.1:18443/
+   # set bigger buffer, since we deploy 50MB WAR
+   SSLRenegBufferSize 52428800
```

Comment:
Please append also commented lines, so we can just fill deploy server hostname.
```
   # restrict access to our deploy server only
   #Order deny,allow
   #Deny from all
   #Allow from your.deploy.server.com
```

Reply:
Appended.